src/sys/inotify.rs

```diff
@@ -5,17 +5,17 @@
 //!
 //! Inotify is a Linux-only API to monitor filesystems events.
 //!
-//! For more documentation, please read [the inotify(7) man page.](http://man7.org/linux/man-pages/man7/inotify.7.html)
+//! For more documentation, please read [inotify(7).](http://man7.org/linux/man-pages/man7/inotify.7.html)
 //! 
 //! Example
 //! =======
 //!
 //! * Monitor all events happening in directory "test":
 //! ```rust,no_run
-//! use nix::sys::inotify::{EventFlags,InotifyInitFlags,InotifyInstance}
+//! use nix::sys::inotify::{EventFlags,InotifyInitFlags,Inotify};
```

Comment:
In doc tests, you can prefix the use lines with `#`.  That way they won't show up in the formatted docs.